packages/documentation/pages/usage/layouts/user-menu.vue

```diff
@@ -22,13 +22,6 @@
 | `userAvatarSrc` | user avatar image src | `String` | —               | —                |
 | `userName`      | user name text        | `String` | —               | —                |
 | `userStatus`    | status of user menu   | `String` | —               | —                |
-
-### Slots
-
-| Slot Name         | Description |
-|:------------------|:------------|
-| `user-menu-items` | menu list   |
```

Comment:
`KtUserMenu`'s Meta does not document this slot

Reply:
I’ve actually removed it in the other PR. It was entirely unused

Reply:
fair enough then